util/cops/deprecations.rb

```diff
@@ -8,42 +8,26 @@ module Rubygems
       #
       # @example
       #
-      #   As of March, 2019
-      #
       #   # bad
-      #   deprecate :safdfa, nil, 2018, 12
-      #   deprecate :safdfa, nil, 2019, 03
+      #   deprecate :safdfa, :none
       #
       #   # good
-      #   deprecate :safdfa, nil, 2019, 04
+      #   the `deprecate` call is fully removed
```

Comment:
I think generally in rubocop's good and bad examples, the code right below the `# good` and `# bad` headers, at the same indentation level, should be valid ruby. So I'd suggest:

```suggestion
      #   # the `deprecate` call is fully removed
```

Reply:
I know that's not what I previously suggested, I just noticed the missing comment when I saw the updated code.